garage/tf/algos/npo.py

```diff
@@ -300,6 +301,10 @@ def _build_policy_loss(self, i):
                         policy_dist_info,
                         name="lr")
 
+                    ll = pol_dist.log_likelihood_sym(
```

Comment:
seems like the scope above should now just be "loss" since it's not necessarily surrogate